js/about.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
```

Comment:
- [optional] Please it is advisable that we should not disable linters, I believe you can resolve those two issues without disabling linters. 👍